pkg/controllers/disruption/suite_test.go

```diff
@@ -150,6 +150,82 @@ var _ = AfterEach(func() {
 	ExpectCleanedUp(ctx, env.Client)
 })
 
+var _ = Describe("Simulate Scheduling", func() {
```

Comment:
I think this test is definitely good. Is there room for ensuring that we can execute concurrent replace requests with our budgets by checking that we don't block on deleting pods scheduling to uninitialized nodes as well?

Reply:
Yeah that's a good idea